mm2src/mm2_db/src/indexed_db/drivers/cursor/cursor.rs

```diff
@@ -242,19 +255,27 @@ impl CursorDriver {
         })
     }
 
-    pub(crate) async fn next(&mut self, where_: Option<CursorCondition>) -> CursorResult<Option<(ItemId, Json)>> {
+    /// Continuously processes cursor items until it retrieves a valid result or
+    /// the cursor is stopped. It returns a `CursorResult` containing either the next item
+    /// wrapped in `Some`, or `None` if the cursor is stopped.
+    pub(crate) async fn next(&mut self) -> CursorResult<Option<(ItemId, Json)>> {
         loop {
             if self.stopped {
                 return Ok(None);
             }
 
-            match self.process_cursor_item(where_.as_ref()).await? {
+            match self.process_cursor_item().await? {
                 Some(result) => return Ok(Some(result)),
                 None => continue,
             }
         }
     }
 
+    /// Continues the cursor according to the provided `CursorAction`. If the action
+    /// is `CursorAction::Continue`, the cursor advances to the next item. If the action is
+    /// `CursorAction::ContinueWithValue`, the cursor advances to the specified value. If the
+    /// action is `CursorAction::Stop`, the cursor is stopped, and subsequent calls to `next`
+    /// will return `None`.
```

Comment:
Could you reformat this explanation into multiple lines (per line for each action)?